apps/docs/app/ui/components/previewComponent/CodeWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import dynamic from "next/dynamic";
+import clsx from "clsx";
+
+const CodeWrapper = async ({ name, key, className }: { name: string; key: string; className?: string }) => {
```

Comment:
This comment will be repeated a couple of time:

- Create an interface instead of inline types.
- is the key prop really necessary?
- clsx(classname) should be replaced by className={className}